### ISSUE TITLE
wa-dropdown keyboard navigation doesn't work with slotted wa-dropdown-item

### ISSUE THREAD

**Issue by medmunds** [shoelace-style/webawesome#1271]
### Describe the bug
If a wa-dropdown is rendered with a `<slot></slot>` child, any wa-dropdown-item elements that are slotted are not accessible using the keyboard.

### To Reproduce

1. Go to https://codepen.io/mvedmunds/pen/qEOmGoe?editors=0010
2. Click on the wa-dropdown
3. Try to use the arrow keys to select one of the dropdown items below the divider
Expected: You should be able to arrow through the entire dropdown
Actual: Only the direct wa-dropdown-item children (above the divider) are accessible with the keyboard

### Demo

https://codepen.io/mvedmunds/pen/qEOmGoe?editors=0010

### Additional information

3.0.0.beta-4

Worked as expected with the equivalent Shoelace components.

**Comment by claviska** [comment]
Thanks for the repro. This has been fixed in #1293 and will be in the next beta release.
